config/features/config.go

```diff
@@ -74,8 +74,9 @@ type Flags struct {
 	CorrectlyInsertOrphanedAtts bool
 	CorrectlyPruneCanonicalAtts bool
 
-	EnableNativeState   bool // EnableNativeState defines whether the beacon state will be represented as a pure Go struct or a Go struct that wraps a proto struct.
-	EnableVectorizedHTR bool // EnableVectorizedHTR specifies whether the beacon state will use the optimized sha256 routines.
+	EnableNativeState    bool // EnableNativeState defines whether the beacon state will be represented as a pure Go struct or a Go struct that wraps a proto struct.
+	EnableVectorizedHTR  bool // EnableVectorizedHTR specifies whether the beacon state will use the optimized sha256 routines.
+	EnableForkChoiceTree bool // EnableForkChoiceTree specifies whether fork choice store will use a doubly linked tree.
```

Comment:
Better to be more specific

```suggestion
	EnableForkChoiceDoubleLinkedTree bool // EnableForkChoiceTree specifies whether fork choice store will use a doubly linked tree.
```

Reply:
nice this was my first flag, but it didn't enter in my terminal :)